docs/settingUp.md

```diff
@@ -32,12 +40,16 @@ More information can be found at [this documentation](https://help.github.com/ar
    ```sh
    ./gradlew appengineDownloadSdk
    ```
+   
+ **Verification** : Check the gradle folder for the existence of the SDK.
```

Comment:
@chowyb I thought if i mention everything wouldn't it be kind of spoon feeding ?.But i will add it.